docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+
+To start with you will use a [dev descriptor](https://developer.shotgunsoftware.com/tk-core/descriptor.html#pointing-to-a-path-on-disk) for the location of your app.
+
+```yaml
+tk-multi-starterapp:
+  location:
+    type: dev
+    path: /path/to/source_code/tk-multi-starterapp
+```
+This instructs Toolkit to load the app code directly from disk in the given location, which is great for development, where you want to change the code all the time.
+Later when you add the app to your production config, you may want to use a different descriptor.
+
+Now you've added the app to an environment, you should be able to go ahead and launch it. How you launch it will depend on the engine and environment you defined it in.
+
+## Part 4: Developing the app
+
+You are now all set to start developing your app!
+
+### Anatomy of the Template Starter App
+
+The [template starter app](https://github.com/shotgunsoftware/tk-multi-starterapp) contains the following basic structure:
+
+![](./images/tk-multi-starterapp-directory.png)
+
+- **app.py** - The app entry point and menu registration can be found in the `app.py` file. This is where you typically set up your classes and get things initialized and get menu items registered.
+- **info.yml** - Also known as the manifest file, defines all the different settings that this app requires when it is installed. 
+Settings are often useful if you want reusable apps and you don't want to hard code any values in the app itself.
+- **python/app/dialog.py** - This contains the logic and event callbacks etc. that produce the main app window.
+- **python/app/ui** - This folder contains the automatically generated UI code and resource file. You don't edit this directly, instead, you edit the Qt UI file in the resources folder.
+- **resources/** - In the resources folder, the `dialog.ui` file is a Qt Designer file that you can open up and use to rapidly design and define the look and feel of the app. 
+Once you have made changes you have to execute the `build_resources.sh` script which will convert the UI file to python code and store it as the `/python/app/ui/dialo.py`.
+- **style.qss** - You can define QSS (Qt style sheets) for your UI in this file.
+
+{% include info title="Note" content="An app doesn't need to have a UI however, and the minimum requirements for a valid app are an `app.py` containing an `Application` class and an `info.yml`." %}
+
+### Configuration settings
+Inside the manifest file, there should be a `configuration` section where you can define app settings.
+
+Defining a setting in the manifest file, allows you to configure different setting values for your app in the environment YAML files.
```

Comment:
no comma in this sentence (sorry!)

Reply:
you're not sorry, are you, otherwise, you wouldn't say these things! 😝